idea/src/org/jetbrains/kotlin/idea/inspections/collections/ConvertCallChainIntoSequenceInspection.kt

```diff
@@ -296,6 +296,7 @@ private val terminations = listOf(
     "singleOrNull",
     "sum",
     "sumBy",
+    "sumByLong",
```

Comment:
I guess all these call are intentionally alphabetically ordered, thus `sumByLong` should go below `sumByDouble`